### DISCUSSION TITLE
Allow sorting reverse dependencies

### DISCUSSION THREAD

**Discussion by matthew-piziak** [rust-lang/crates.io#4347] [Feature Requests]
`/search` accepts query arguments `sort=downloads` and `sort=relevance`.

`/crates/{crate}/reverse_dependencies` does not accept a `sort` query parameter. All reverse dependencies are in alphabetical order

"What are the most popular crates that use this dependency" is a valuable use case. For example, today I wanted to see good living examples of the `iron` crate, but was unable to distinguish them from other crates through the crates.io interface.

**Comment by carols10cents** [comment]
Sooo reverse dependencies are now ordered by downloads by default, but there still isn't a sort option, so now you can't sort alphabetically, so this bug still stands in a way :)

**Comment by matthew-piziak** [comment]
Thanks for doing this! :)
